sync/import-export-dropbox.js

```diff
@@ -0,0 +1,179 @@
+/* global messageBox Dropbox createZipFileFromText readZipFileFromBlob launchWebAuthFlow getRedirectUrlAuthFlow importFromString resolve */
+'use strict';
+
+const DROPBOX_API_KEY = 'zg52vphuapvpng9';
+const FILENAME_ZIP_FILE = 'stylus.json';
+const DROPBOX_FILE = 'stylus.zip';
+const API_ERROR_STATUS_FILE_NOT_FOUND = 409;
+const HTTP_STATUS_CANCEL = 499;
+
+function messageProgressBar(data) {
+  return messageBox({
+    title: `${data.title}`,
+    className: 'config-dialog',
+    contents: [
+      $create('p', data.text)
+    ],
+    buttons: [{
+      textContent: t('confirmClose'),
+      dataset: {cmd: 'close'},
+    }],
+  }).then(() => {
+    document.body.style.minWidth = '';
+    document.body.style.minHeight = '';
+  });
+}
+
+function hasDropboxAccessToken() {
+  return chromeLocal.getValue('dropbox_access_token');
+}
+
+function requestDropboxAccessToken() {
+  const client = new Dropbox.Dropbox({clientId: DROPBOX_API_KEY});
+  const authUrl = client.getAuthenticationUrl(getRedirectUrlAuthFlow());
+
+  return launchWebAuthFlow({url: authUrl, interactive: true})
+  .then(urlReturned => {
+    const params = new URLSearchParams(new URL(urlReturned).hash.replace('#', ''));
+    chromeLocal.setValue('dropbox_access_token', params.get('access_token'));
+    return params.get('access_token');
+  });
+}
+
+function uploadFileDropbox(client, stylesText) {
+  return client.filesUpload({path: '/' + DROPBOX_FILE, contents: stylesText});
+}
+
+$('#sync-dropbox-export').onclick = () => {
+  const title = t('syncDropboxStyles');
+  messageProgressBar({title: title, text: t('connectingDropbox')});
+
+  hasDropboxAccessToken()
+  .then(token => token || requestDropboxAccessToken())
+  .then(token => {
+    const client = new Dropbox.Dropbox({
+      clientId: DROPBOX_API_KEY,
+      accessToken: token
+    });
+
```

Comment:
These empty lines before `return` visually regroup the code in the wrong way - separating each .then block in the middle. Please remove them everywhere.

Reply:
ok ^^

Reply:
done :)